translations/ui-data-import/en_GB.json

```diff
@@ -17,6 +17,7 @@
     "jobProfileName": "Job profile",
     "jobExecutionHrId": "ID",
     "jobCompletedDate": "Ended running",
+    "jobParts": "Job parts",
```

Comment:
Do these do anything? I was under the impression that `en.json` was the only one that mattered and the others were all handled by Lokalise

Reply:
en_GB may not... let me remove it and try... en_US may be the only one that gets used here.

Reply:
i think it gets used in local stripes dev, but i thought it was overwrited with translation files — was just curious, no worries!